### ISSUE TITLE
can't open file 'find_stable.py': [Errno 2] No such file or directory

### ISSUE THREAD

**Issue by DeletedUser-0** [Ben-0-mad/YT-TMS-Finder#9]
Hello, I hope you are having a great day.
When I type "python find_stable.py", it isn't working, appearing `python: can't open file 'find_stable.py': [Errno 2] No such file or directory`.
I have python, yeah, version 3.8.5, but my command prompt (even executing with cmd) isn't identifying that prefix. 
I have followed all the instructions.
How can it be fixed?

**Comment by Ben-0-mad** [comment]
Hi, your python installation is fine. All you have to do is navigate to the YT-TMS-Finder folder that you just downloaded. If you do the "dir" command, it lists all the files in the directory you are in, you should also see "find_stable.py". If you don't see it, you're not in the right folder. Hope this helps!

**Comment by DeletedUser-0** [comment]
thanks for your help!

**Comment by DeletedUser-0** [comment]
oh i am getting an error! `Traceback (most recent call last):
  File "find_stable.py", line 3, in <module>
    from selenium import webdriver          #for getting html source of channel
ModuleNotFoundError: No module named 'selenium'`

**Comment by Ben-0-mad** [comment]
No problem 👍🏽 if you experience any other problems pease let me know. I will close this issue now.

**Comment by DeletedUser-0** [comment]
But that means that I can't have access to find "The Mysterious Song"!

**Comment by Ben-0-mad** [comment]
Did you get the latest release and follow the setup for windows?

In step 2 it says ```pip install -r requirements.txt```, this will install selenium and other needed modules.

And then you also have to follow the other steps of the installation.

**Comment by DeletedUser-0** [comment]
Yes, I did, like I said in my first message.

**Comment by Ben-0-mad** [comment]
Tl;dr : do ```pip install selenium```


Can you tell me which version of YT-TMS-Finder you are using? If you're using an earlier version I used webbot for that so it will install webbot instead of selenium. In the newer versions, I used selenium and added that to the requirements.txt.

Alternatively you can install the module by doing ```pip install selenium```
